numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -3,6 +3,9 @@
 from numba.cuda.cudadrv.error import (NvrtcError, NvrtcCompilationError,
                                       NvrtcSupportError)
 from numba.cuda.cuda_paths import get_cuda_paths
+
+from .info import CTK_SUPPORTED
```

Comment:
It looks like this is not aligned with the rest of the file. I think we're preferring full import paths like `from numba.cuda ...` instead of importing from the local module structure.

However, I see that `nvvm.py` has a different style. I guess I would recommend being consistent within each file.

```suggestion
from numba.cuda.cudadrv.info import CTK_SUPPORTED
```

Reply:
I agree that we should stay consistent with module import patterns.